app/actions/wallets-actions.js

```diff
@@ -4,7 +4,7 @@ import Action from './lib/Action';
 // ======= WALLET ACTIONS =======
 
 export default class WalletsActions {
-  createWallet: Action<{ name: string, currency: string }> = new Action();
+  createWallet: Action<{ name: string, currency: string, password: string }> = new Action();
```

Comment:
I guess you should make the password field optional like this `password?: string` (means the property can be missing) or like this: `password: ?string` (means it can either be a string or null)